autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py

```diff
@@ -57,6 +57,16 @@ def setUpClass(cls):  # nopep8 - ignore N802
         """
         super(AutoscaleRackConnectFixture, cls).setUpClass()
 
+        if cls.rcv3_client is None:
+            # don't bother with the rest of the set-up; skip all tests
+            skip = unittest.skip("This account does not support RCv3.")
+
+            for test in inspect.getmembers(cls, predicate=inspect.ismethod):
+                # test is a tuple of name, object
+                if test[0].startswith("test_"):
```

Comment:
This is an interesting trick to skip tests at runtime. I wonder if there is another cleaner way to do the same. This is merely non-blocking thought :)